src/components/result/ButtonResult.tsx

```diff
@@ -12,16 +13,37 @@ type PropsType = {
 const ButtonResult = (props: PropsType): JSX.Element => {
     const { config } = props;
 
+    const score = useStoreSelector((state) => state.result.score);
+    const answers = useStoreSelector((state) => state.answers.list);
     const dispatch = useStoreDispatch();
 
-    const restartAnswers = (): void => {
-        dispatch(resetAnswers());
-        dispatch(showResult(false));
+    const onClick = (): void => {
+        const url = replaceValues(config.url, { score })!;
+        switch (config.function) {
+            case "restart":
+                window.scrollTo({ top: 0 });
+                dispatch(showResult(false));
+                dispatch(resetAnswers());
+                break;
+            case "postData":
+                fetchAnswerData(config.url as AnswerDataUrl, {
+                    score: score,
+                    answers: answers,
+                });
```

Comment:
Can become a oneliner like
```suggestion
                fetchAnswerData(config.url as AnswerDataUrl, { score, answers });
```